api/services/ocean/spark/schemas/oceanSparkApplication.yaml

```diff
@@ -0,0 +1,160 @@
+      $ref: "oceanSparkApplicationId.yaml"
+    displayName:
+      $ref: "oceanSparkApplicationDisplayName.yaml"
+    accountId:
+      $ref: "accountId.yaml"
+    organizationId:
+      $ref: "organizationId.yaml"
+    userId:
+      $ref: "userId.yaml"
+    clusterId:
+      $ref: "oceanSparkClusterId.yaml"
+    controllerClusterId:
+      $ref: "controllerClusterId.yaml"
+    appState:
+      $ref: "oceanSparkApplicationState.yaml"
+    submissionSource:
+      $ref: "oceanSparkApplicationSubmissionSource.yaml"
+    job:
+      type: object
+      description: >
+        The job to which the application belongs.
+        A job is collection of Spark applications, typically a workload run on a schedule.
+      required:
+        - id
+        - displayName
+      properties:
+        id:
+          $ref: oceanSparkJobId.yaml
+        displayName:
+          $ref: oceanSparkJobDisplayName.yaml
+    config:
+      $ref: "oceanSparkApplicationSpec.yaml"
+    startedAt:
+      type: string
+      format: date-time
+      description: >
+        The time when the Spark application started running (the app state becomes RUNNING).
+        This is not the time when the application launch was requested (this is `createdAt`).
+      example: "2021-11-18T17:09:37+00:00"
+    endedAt:
+      type: string
+      format: date-time
+      description: >
+        The time when the Spark application terminated.
+      example: "2021-11-18T17:09:37+00:00"
+    log:
+      type: object
+      description: API pointers to endpoints serving logs and events related to the Spark application
+      properties:
+        logsStreamUrl:
+          type: string
+          format: uri
+          description: URL of the endpoint serving the live driver logs
+          example: /ocean/spark/cluster/osc-20fac3f1/app/daily-reporting-2021-08-18/logs/live
+        kubeEventsStreamUrl:
+          type: string
+          format: uri
+          description: URL of the endpoint serving a stream of Kubernetes events
+          example: /ocean/spark/cluster/osc-20fac3f1/app/daily-reporting-2021-08-18/kubeEvents/live
+    metrics:
+      type: object
+      description: Metrics about the Spark application
+      properties:
+        cost:
+          type: object
+          required:
+          - nonPvStorage
+          - storage
+          - compute
+          - total
+          - createdAt
+          - updatedAt
+          description: The cloud cost of the Spark application. Updated every hour
+          properties:
+            createdAt:
+              # I had to write createdAt and updatedAt manually and not use createdAtUpdatedAt.yaml.
+              # Otherwise the object is marked as "recursive". This looks like a redoc bug.
+              $ref: "createdAt.yaml"
+            updatedAt:
+              $ref: "updatedAt.yaml"
+            nonPvStorage:
+              type: integer
+              description: The fraction of the cost associated to ephemeral storage in cents
```

Comment:
Rather than "fraction" (which sounds like it would be a percentage), I would describe this as "portion".

Same comment for the other cost fields below.

Also I would say "USD cents".

Reply:
Thanks!